.github/ISSUE_TEMPLATE/new-event-fellowship-round.md

```diff
@@ -0,0 +1,60 @@
+---
+name: New Round for OBF Event Fellowship Application
+about: 'This template is for the OBF Event Fellowship chair to manage a new application round'
+
+---
+
+**OBF Event Fellowship Round:**
+
+- [ ] Round-1: 15 May
+- [ ] Round-2: 15 September
+- [ ] Round-3: 15 January
```

Comment:
```suggestion
- [ ] Round-1: 15 May
- [ ] Round-2: 15 September
- [ ] Round-3: 15 January
```
+ [ ] Round 1: 1 October (2022 only)
+ [ ] Round 2: 1 February
+ [ ] Round 3: 1 May (for 2022 -- may need to be earlier in 2023 because ISMB/BOSC will be earlier)
+ [ ] Round 4: 1 August
+ [ ] Round 1 (starting in 2023, instead of 1 October): 1 November

Reply:
(I hope I did that Suggesting block right -- I've forgotten how to add the suggested changes!)

Reply:
Yes